lib/ansible/modules/system/known_hosts.py

```diff
@@ -323,20 +322,21 @@ def compute_diff(path, found_line, replace_or_add, state, key):
     diff['after'] = ''.join(lines)
     return diff
 
+
 def main():
 
     module = AnsibleModule(
-        argument_spec = dict(
-            name      = dict(required=True,  type='str', aliases=['host']),
-            key       = dict(required=False,  type='str'),
-            path      = dict(default="~/.ssh/known_hosts", type='path'),
-            hash_host = dict(required=False, type='bool' ,default=False),
-            state     = dict(default='present', choices=['absent','present']),
-            ),
-        supports_check_mode = True
-        )
-
-    results = enforce_state(module,module.params)
+        argument_spec=dict(
```

Comment:
No, the previous formatting was much better here for this block, much more readable.